network/service/handler/handler.go

```diff
@@ -94,11 +94,6 @@ func (n *Network) Connect(ctx context.Context, req *pbNet.ConnectRequest, resp *
 
 // Nodes returns the list of nodes
 func (n *Network) Nodes(ctx context.Context, req *pbNet.NodesRequest, resp *pbNet.NodesResponse) error {
-	depth := uint(req.Depth)
-	if depth <= 0 || depth > network.MaxDepth {
```

Comment:
why this removed?
first check <= 0 is redundant because of uint before, but why removed MaxDepth checking?

Reply:
but depth seems don'nt be used.

Reply:
Yeah, this seems to be a copy-paste leftover and should be removed completely.

Reply:
indeed, depth is not used

Reply:
nice